lib/SearchAndSort/SearchAndSort.js

```diff
@@ -364,12 +367,13 @@ class SearchAndSort extends React.Component {
 
   onFilterChangeHandler = (filter) => {
     const {
-      parentMutator: { resultCount },
+      parentMutator: { resultCount, resultOffset },
       initialResultCount,
       onFilterChange,
     } = this.props;
 
     resultCount.replace(initialResultCount);
+    if (resultOffset) resultOffset.replace(0);
```

Comment:
This isn't wrong, but personally, I strongly prefer that all conditionals be written with `{}` across multiple lines. It's so easy to indent something under this line and think it's part of the condition when it is not. Defensive programming and all that. 

TBH, given all the stooooooooooooooopid AirBNB eslint rules we put up with, I'm astonished this practice isn't among them.

Reply:
Fair enough. I'll fix them.